tests/kernel/common/src/irq_offload.c

```diff
@@ -95,7 +95,7 @@ void test_irq_offload(void)
  *
  * @see arch_nop()
  */
-__no_optimization void test_nop(void)
+__no_optimization void nop(void)
```

Comment:
I am not sure how to apply `__no_optimization` in new ztest API,
@enjiamai , is it ok for you to change like this?

Reply:
sure, no problem, actually, I am planning to remove this test_nop testcase, it is not very good. :)